x-pack/plugins/observability/public/components/shared/core_web_vitals/service_name.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { EuiIcon, EuiText, EuiTitle, EuiToolTip } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { i18n } from '@kbn/i18n';
+
+interface Props {
+  name: string;
+}
+
+const SERVICE_LABEL = i18n.translate('xpack.observability.ux.coreWebVitals.service', {
+  defaultMessage: 'Service',
+});
+
+export function ServiceName({ name }: Props) {
+  return (
+    <>
+      <EuiText>
+        {SERVICE_LABEL}
+        <EuiToolTip
+          content={
+            <EuiText>
+              <FormattedMessage
+                id="xpack.observability.ux.service.help"
+                defaultMessage="Most traffic"
+              />
+            </EuiText>
+          }
+        >
+          <EuiIcon color={'text'} type={'questionInCircle'} />
+        </EuiToolTip>
```

Comment:
You can use the [EuiIconTip](https://elastic.github.io/eui/#/display/tooltip#icontip) instead of creating this custom tooltip.

Reply:
done in [`16104e8` (#78976)](https://github.com/elastic/kibana/pull/78976/commits/16104e8d3d4f4661bc385497e26b8a33a9983647)